federation/planner.go

```diff
@@ -145,22 +192,34 @@ func (e *Planner) planObject(typ *graphql.Object, selectionSet *graphql.Selectio
 			return nil, fmt.Errorf("typ %s has no field %s", typ.Name, selection.Name)
 		}
 
-		fieldInfo := e.schema.Fields[field]
-
-		// Prioritize resolving as many fields as we can in the current service
-		if fieldInfo.Services[service] {
-			localSelections = append(localSelections, selection)
-		} else {
-			serviceWithField := ""
+		var hasCustomService bool
+		hasCustomService, localSelections, selectionsByService = e.maybeCustomService(
```

Comment:
What do you think about changing`maybeCustomService` to `isMigratingField` to make it more clear that this is only true in the case of a field migration. 

Custom service seems kinda confusing to me since if you're moving it from one team's server to a smaller subteams server, you're moving to from one custom server to another custom server. I seem to think of all the federated servers as custom servers, but I want the naming to tell me that this is specifically used for migrating a field.

Reply:
sg, changed the name.